tests/test_multigrid.py

```diff
@@ -0,0 +1,76 @@
+from parcels import FieldSet, Field, ParticleSet, StructuredGrid, ScipyParticle, JITParticle, Variable
+from parcels import AdvectionRK4
+import numpy as np
+import pytest
```

Comment:
Not sure what the purpose of this `test_multigrid.py` function is. It looks more like an `example` than a short unit-test? 
And if we want to keep it as a unit-test, then why not within `test_fieldset_sampling`?